### PR TITLE
Ru ref and survey id on contact page

### PR DESCRIPTION
**Pull request by MIslam92**
### What is the context of this PR?
Now passes though the Survey_ID and Ru_Ref to the Contact page and when respondent wants to email passes the Ru_Ref to the subject line of the email.

Card: https://trello.com/c/qj51cXTL/1422-ru-ref-and-survey-id-on-contact-page-s

### How to review 
Open a survey and go to Contact Us page to check if they appear.
Also, when you click on the email link the Ru_Ref is passed into the email subject.

### CODE REVIEW

File: tests/integration/views/test_static.py
Comment (jonnyshaw89) [inline tests/integration/views/test_static.py]: These asserts are not specific enough as these values are in the footer, even on the error pages

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Comment by shahi645** [comment]
@MIslam92 Just thinking do we track analytics on this, if so does it track the url ?

**Comment by MIslam92** [comment]
@shahi645 it should but does not at the moment. Going to add the tags for them

**Comment by shahi645** [comment]
ill wait for changes and re-review

File: app/templates/static/contact-us.html
Comment (shahi645) [inline app/templates/static/contact-us.html]: this div should be indented to, but being picky

File: app/templates/static/contact-us.html
Comment (jonnyshaw89) [inline app/templates/static/contact-us.html]: shouldn't this be `<br />`?

File: app/templates/static/contact-us.html
Comment (jonnyshaw89) [inline app/templates/static/contact-us.html]: this line shouldn't be indented

File: app/views/static.py
Comment (jonnyshaw89) [inline app/views/static.py]: if you use keyword arguments then you should add the keyword to all of the arguments. in this instance the 1st argument doesn't have a keyword

File: tests/integration/views/test_static.py
Comment (jonnyshaw89) [inline tests/integration/views/test_static.py]: Don't we want to be asserting that the ru ref is on the page?

File: tests/integration/views/test_static.py
Comment (jonnyshaw89) [inline tests/integration/views/test_static.py]: this page shouldn't error should it?

File: tests/integration/views/test_static.py
Comment (jonnyshaw89) [inline tests/integration/views/test_static.py]: We need to check a larger section of the page

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: tests/integration/views/test_static.py
Comment (MIslam92) [reply tests/integration/views/test_static.py]: @jonnyshaw89 This should error as they would only see that page if they are an authenticated user i.e. logged in

Comment (jonnyshaw89) [reply tests/integration/views/test_static.py]: @MIslam92 i have just confirmed the behaviour with @charlotte-hirst.
When not logged in the behaviour should be that "two lines of text are not there "Please quote ..." and "Your survey id ..""
The card will be updated with this.

File: app/views/static.py
Comment (ajmaddaford) [inline app/views/static.py]: We should be getting this information from the session, rather than metadata. The metadata is removed on submission, the session will still hold the eq_id, form_type and ru_ref.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: app/views/static.py
Comment (dwyeradam) [inline app/views/static.py]: Session data is already held in the session_store which you can get from globals.py (get_session_store). questionnaire.py uses this if you'd like an example. You can then get the schema using the eq-id and form_type from the session data.

File: app/views/static.py
Comment (jonnyshaw89) [inline app/views/static.py]: This is just going to add to #1442

File: tests/integration/views/test_static.py
Comment (jonnyshaw89) [inline tests/integration/views/test_static.py]: This shouldn't error. it should show the page without the details from the session

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Review by dwyeradam — CHANGES_REQUESTED** [review]
If you save and sign out and then click the contact us link you get a 500 error.

File: tests/integration/views/test_static.py
Comment (dwyeradam) [inline tests/integration/views/test_static.py]: This test is no longer valid and the new tests should cover whether session_data is available or not.

**Review by dwyeradam — CHANGES_REQUESTED** [review]

File: app/templates/static/contact-us.html
Comment (shahi645) [inline app/templates/static/contact-us.html]: spec asks for line breaks instead of paragraph tags

**Review by shahi645 — DISMISSED** [review]

File: app/templates/static/contact-us.html
Comment (shahi645) [reply app/templates/static/contact-us.html]: Also `Opening hours` Should be h3 'Venus'

**Comment by shahi645** [comment]
@MIslam92 Still missing ru ref in subject of email and also banner content need to change

File: tests/integration/views/test_static.py
Comment (dwyeradam) [inline tests/integration/views/test_static.py]: You've just changed the name. This isn't checking the differences on the page when session_data is available compared to when it's not. This test would pass with or without session_data.

File: tests/integration/views/test_static.py
Comment (dwyeradam) [inline tests/integration/views/test_static.py]: You've just changed the name. This isn't checking the differences on the page when session_data is available compared to when it's not. This test would pass with or without session_data.

**Review by dwyeradam — CHANGES_REQUESTED** [review]

File: tests/integration/views/test_static.py
Comment (jonnyshaw89) [inline tests/integration/views/test_static.py:20]: You also need to add a positive assert as this would not be there on an error page

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: app/views/static.py
Comment (dwyeradam) [inline app/views/static.py:22]: This will fail with surveys launched from URL. Once merged i'll fix it as part of #1450.

File: app/templates/static/contact-us.html
Comment (jonnyshaw89) [inline app/templates/static/contact-us.html]: should we have a `h3` in a `p`?

Comment (jonnyshaw89) [reply app/templates/static/contact-us.html]: i think the `p` should wrap the next 2 lines

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Review by dwyeradam — DISMISSED** [review]

File: app/templates/static/contact-us.html
Comment (jonnyshaw89) [inline app/templates/static/contact-us.html]: This `if` breaks the `p` tag nesting.
the `p` tags should be inside the `if`

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: app/templates/static/contact-us.html
Comment (dwyeradam) [inline app/templates/static/contact-us.html]: This just doesn't look right over 3 lines. Also the  `p` should be nested inside the if statement.

File: app/templates/static/contact-us.html
Comment (dwyeradam) [inline app/templates/static/contact-us.html]: Can you move the line breaks to the end of the line.
`Office for National Statistics<br>`
`Government Buildings<br>`

**Review by dwyeradam — CHANGES_REQUESTED** [review]

File: app/templates/static/contact-us.html
Comment (jonnyshaw89) [inline app/templates/static/contact-us.html]: I don't think you need the trailing line break

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Review by jonnyshaw89 — APPROVED** [review]

**Review by dwyeradam — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**
